cetmix_tower_server/models/cx_tower_reference_variable_option.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ReferencesVariableOption(models.Model):
+    _name = "cx.tower.reference.variable.option"
+    _inherit = ["cx.tower.reference.mixin"]
+
+    name = fields.Char(related="variable_id.name", required=True)
+    reference = fields.Char(related="variable_id.reference", readonly=True)
```

Comment:
According to the task: `Variable Option Reference = Variable  Reference + "option" + Option Reference.`